### ISSUE TITLE
About the output data of the LSD line detector

### ISSUE THREAD

**Issue by yiyihao2000** [yosungho/LineTR#19]
Hello, your work is excellent I have a question to ask you, which of the line segment feature data output by the line detector used by the line transformer? For example, what information is there beside the starting point, ending point, and angle of the line segment? Thanks

**Comment by yosungho** [comment]
Hi, the linetr needs two endpoints from a line detector, and its length, angle, and sample points between the endpoints are used in the networks.

**Comment by yiyihao2000** [comment]
Thank you for your answer! May I ask if the two endpoints you mentioned refer to the starting and ending points of a line segment? Also, is the selection of sampling points related to line detectors?

**Comment by yosungho** [comment]
Yes. The two endpoints are the starting and ending points. The point samples on a line do not depend on the type of line detector. They are just the equi-distanced points between the two endpoints.

**Comment by yiyihao2000** [comment]
> 是的。这两个端点是起点和终点。线上的点采样不依赖于线检测器的类型。它们只是两个端点之间的等距点。
Thank you for your answer. I have another question that I would like to trouble you with. May I ask what dimension tuple array is the output "klines_cv2" of the LSD line detector?

**Comment by yiyihao2000** [comment]
Thank you for your answer. I have another question that I would like to trouble you with. May I ask what dimension tuple array is the output "klines_cv2" of the LSD line detector?
